src/tindev-commits.html

```diff
@@ -0,0 +1,62 @@
+<link rel="import" async href="../bower_components/polymer/polymer.html">
+<link rel="import" async href="../bower_components/iron-ajax/iron-ajax.html">
+<link rel="import" async href="../bower_components/code-highlighter/code-highlighter.html">
+<link rel="import" async href="../bower_components/marked-element/marked-element.html">
+
+<dom-module id="tindev-commits">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+
+      img {
+        float: left;
+        width: 200px;
+        margin: 0 10px 10px 0;
+      }
+
+      .code {
+        clear: left;
+      }
+    </style>
+    <iron-ajax
+      auto
+      url="https://api.github.com/repos/polymer/polymer/commits/3d054bab6645421878598d26498a9f2a47804b5d"
```

Comment:
Yes, for now. Still waiting on @mpasterkamp to finish the import, such that we can hook `es-query` into it.